CONTRIBUTING.md

```diff
@@ -29,9 +29,15 @@ run the tests, it can be useful to prepare a
 [virtual environment](https://docs.python.org/3/tutorial/venv.html) and install
 certain libraries typeshed uses to check stub files.
 
-Follow platform-specific instructions below. Following that, to automatically
-check your code before committing, you can copy the file `pre-commit` to
-`.git/hooks/pre-commit`.
+Follow platform-specific instructions below.
+
+The repository is equipped with a [`pre-commit`](https://pre-commit.com/)
```

Comment:
It would be nicer to say `pre-commit.ci`, to make it clear that you don't have to run anything locally.

Reply:
Done